src/components/calcite-block/calcite-block.scss

```diff
@@ -160,6 +160,9 @@ calcite-action-menu {
   .icon {
     @apply ml-0 mr-3;
   }
+  .toggle {
+    @apply text-right;
```

Comment:
Hmmm...I think as @driskull it might be better to take this opportunity to refactor with logical properties for the margins.

For the text, we could change the `text-align` rule [here](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-block/calcite-block.scss#L54) to be `initial`.